### DISCUSSION TITLE
Use GitHub block quotes extension

### DISCUSSION THREAD

**Discussion by uhthomas** [renovatebot/renovate#24549] [Suggest an Idea]
### Tell us more.

GitHub has extended the blockquote syntax to include more visible messages. It would be nice if Renovate were to use these instead the existing style.

https://github.com/orgs/community/discussions/16925

Example:

> [!NOTE]  
> Highlights information that users should take into account, even when skimming.

> [!IMPORTANT]  
> Crucial information necessary for users to succeed.

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

Current:

<img width="910" alt="Screenshot 2023-09-20 at 13 19 46" src="https://github.com/renovatebot/renovate/assets/9749173/9f1160ea-d8d7-4bb9-870a-35199585a4d7">

With proposed change:

<img width="911" alt="Screenshot 2023-09-20 at 13 20 29" src="https://github.com/renovatebot/renovate/assets/9749173/6c0fe9f5-b82b-42be-9484-a5f2e25b6846">

**Comment by rarkins** [comment]
Good idea, this can probably be done using the existing `massageMarkdown()` function in platform/github

**Reply by rarkins** [reply]
Created feature request: https://github.com/renovatebot/renovate/issues/24550
